src/main/java/org/opensearch/security/auditlog/impl/AuditMessage.java

```diff
@@ -60,7 +60,12 @@
     private static final WildcardMatcher AUTHORIZATION_HEADER = WildcardMatcher.from("Authorization", false);
     private static final String SENSITIVE_KEY = "password";
     private static final String SENSITIVE_REPLACEMENT_VALUE = "__SENSITIVE__";
-    private static final Pattern SENSITIVE_PATHS = Pattern.compile("/_opendistro/_security/api/(account.*|internalusers.*|user.*)");
+
+    private static final String LEGACY_OPENDISTRO_PREFIX = "_opendistro/_security";
+    private static final String PLUGINS_PREFIX = "_plugins/_security";
```

Comment:
same comment as above. Use utility

Reply:
Same as above

Reply:
As discussed, please keep in check for later improvement

Reply:
Sure